Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/API/wmb.yaml

```diff
@@ -0,0 +1,90 @@
+imports:
+  Text: Data.Text
+  APISuccess: Kernel.Types.APISuccess
+  LatLong: Kernel.External.Maps.Types
+
+module: WMB
+
+types:
+  RouteInfo:
+    code: Text
+    shortName: Text
+    longName: Text
+    startPoint: LatLong
+    endPoint: LatLong
+
+  AvailableRoutesList:
```

Comment:
AvailableRoutesList:
    routeInfo: RouteInfo
    source: StopInfo
    destination: StopInfo

Reply:
send `vehicleDetails` as well in this response